pkg/plugins/mercury/value.go

```diff
@@ -0,0 +1,41 @@
+package mercury
+
+import (
+	"math/big"
+
+	"github.com/smartcontractkit/libocr/bigbigendian"
+)
+
+var i = big.NewInt
+
+// Bounds on an int192
+const byteWidth = 24
+const bitWidth = byteWidth * 8
+
+var one *big.Int = big.NewInt(1)
+
+// 2**191-1
+func MaxValue() *big.Int {
+	result := MinValue()
+	result.Abs(result)
```

Comment:
Why not just `result.Lsh(one, bitWidth-1)` instead of `Abs(Neg(...))` of that?

Reply:
I copied this from libocr :D

Reply:
Its not even used. Removed.